eng/scripts/generate_from_source_pom.py

```diff
@@ -52,6 +52,18 @@
 
 sdk_string = "/sdk/"
 
+# Exclude artifacts list when build from source.These artifacts may be not compatible with current dependencies.
+exclude_artifacts_list = [
```

Comment:
/CC @alzimmermsft @srnagar 
I don't like having this exclude list in here. It seems that if there are compatibility problems that this is just going to hide them when they need to be pointed out and addressed.